src/postgraphile/cli.ts

```diff
@@ -592,7 +661,9 @@ const postgraphileOptions = pluginHook(
     enhanceGraphiql: enhanceGraphiql ? true : undefined,
     jwtPgTypeIdentifier: jwtPgTypeIdentifier || deprecatedJwtPgTypeIdentifier,
     jwtSecret: jwtSecret || deprecatedJwtSecret,
+    jwtPublicKey,
```

Comment:
Do we need a new field for this, or can you just send the public key via `jwtSecret`? (Is this just so people can find it easier?) I feel like passing a public key to `jwtSecret` is fine; it's effectively saying "the JWT secret is identified by this public key". What do you think?

Reply:
The issue is that if Postgraphile issues and verified JWTs by itself, it has to have a pair of priv/pub keys configured. Only when it is used only for issuing JWTs or only for verifying JWTs the single `secret` parameter can be used.

Reply:
One of the options is to deprecate the `jwtSecret` option and propagate users to use `jwtPublicKey` and `jwtPrivateKey` instead. This will remove any misunderstandings in the future. What do you think about it?

Reply:
Maybe `jwtSignKey` and `jwtVerifyKey`?

Reply:
i would vote against having a dual-use-kinda parameter (eg `jwtSecret` means secret for symetric, but public-key for asymmetric)

Even tho it is more code i would vote for something like the following, because IMHO stuff should be explicitly clear, especially when related to security/crypto:

- separate signing and verifying-options (as we might have use-cases where postgraphile just does one of the two)
- have `signingAlgorithm` and `verifyingAlgorithm` options
- have `signingSecret`, `signingPublicKey` and `signingPrivateKey` but make the code check (eg `signingSecret` is only applicable+required for symetric algorithms)

Reply:
@betaboon I think, `jwtSignKey` and `jwtVerifyKey` are the most appropriate names. Because a signing key can be a secret or a private key. But it is used for signing. A verify key can be the same secret or a public key. So, I don't see any issues. Anyway, all of the options: `signingSecret`, `signingPublicKey` and `signingPrivateKey` would be passed to the same parameter to the internal sign function. Separation of these values will lead to bringing a lot of logic about crypto algorithms used by the third-party library. I think it is not a good way in terms of future support to keep the logic in actual state.

Reply:
I think the pairing of jwtSecret (private key, or secret string) and jwtPublicKey works well.